requirements/edx-sandbox/base.in

```diff
@@ -7,5 +7,7 @@
 #   * confirm that it has no system requirements beyond what we already install
 #   * run "make upgrade" to update the detailed requirements files
 
+-c ../constraints.txt
```

Comment:
Hmm, do you think we should call it `constraints.in` instead to make it more explicit that it's an input to `pip-compile`?

Reply:
I thought about that, and went with .txt because we want it to be explicit pins with no further interpretation or resolution. That is, this file is directly usable during the installation phase as well as the resolution phase.  And we already have .txt files included into .in files elsewhere.